contributors.txt

```diff
@@ -0,0 +1,57 @@
+Git Extensions Project Contributors Certification of Origin and Rights
+
+All contributors to Git Extensions must formally agree to abide by this
+certificate of origin by signing on the bottom with their github
+userid, full name, email address (you can obscure your e-mail, but it
+must be computable by human), and date.
+
+By signing this agreement, you are warranting and representing that
+you have the right to release code contributions or other content free
+of any obligations to third parties and are granting Henk Westhuis and
+Git Extensions project contributors, henceforth referred to as 
+The Git Extensions Project, a license to incorporate it into 
+The Git Extensions Project or related works under the MIT license. 
+You understand that The Git Extensions Project may or may not incorporate
+your contribution and you warrant and represent the following:
+
+1. I am the creator of all my contributions. I am the author of all
+   contributed work submitted and further warrant and represent that
+   such work is my original creation and I have the right to license
+   it to The Git Extensions Project for release under the MIT and/or GPL
+   license. I hereby grant The Git Extensions Project a nonexclusive,
+   irrevocable, royalty-free, worldwide license to reproduce,
+   distribute, prepare derivative works, and otherwise use this
+   contribution as part of the Git Extensions project, associated
```

Comment:
💡 ... use ~~this contribution~~ **my contributions** as part ...

Reply:
Thank you